openshift/template_insights-inventory.yaml

```diff
@@ -0,0 +1,183 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: rhsm-insights-inventory
+objects:
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    generation: 1
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}
+  spec:
+    replicas: 1
+    selector:
+      deploymentconfig: ${APPLICATION_NAME}
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        pre:
+          execNewPod:
+            command:
+              - python
+              - manage.py
+              - db
+              - upgrade
+            containerName: ${APPLICATION_NAME}
+          failurePolicy: Abort
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: ${APPLICATION_NAME}
+          deploymentconfig: ${APPLICATION_NAME}
+      spec:
+        containers:
+          - image: ${APPLICATION_NAME}
+            imagePullPolicy: Always
+            name: ${APPLICATION_NAME}
+            ports:
+              - containerPort: 8080
+                protocol: TCP
+        restartPolicy: Always
+        terminationGracePeriodSeconds: 75
+    triggers:
+      - imageChangeParams:
+          automatic: true
+          containerNames:
+            - ${APPLICATION_NAME}
+          from:
+            kind: ImageStreamTag
+            name: ${APPLICATION_NAME}:latest
+        type: ImageChange
+      - type: ConfigChange
+
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${APPLICATION_NAME}
+  spec:
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${APPLICATION_NAME}:latest
+    source:
+      git:
+        ref: ${SOURCE_REPOSITORY_REF}
+        uri: ${SOURCE_REPOSITORY_URL}
+      type: Git
+    strategy:
+      sourceStrategy:
+        env:
+          - name: INVENTORY_DB_HOST
+            value: ${INVENTORY_DB_HOSTNAME}
+          - name: INVENTORY_DB_USER
+            value: ${INVENTORY_DB_USER}
+          - name: INVENTORY_DB_PASS
+            value: ${INVENTORY_DB_PASS}
+          - name: INVENTORY_DB_NAME
+            value: ${INVENTORY_DB_NAME}
+          - name: APP_NAME
+            value: ${INVENTORY_APP_NAME}
+          - name: PATH_PREFIX
+            value: ${INVENTORY_PATH_PREFIX}
+          - name: prometheus_multiproc_dir
+            value: /tmp
```

Comment:
This could use a disc volume and point to something other than /tmp. Forgot to set that up. Will do that soon.